packages/s/startbootstrap-sb-admin-2.json

```
@@ -14,7 +13,7 @@
     "target": "git://github.com/BlackrockDigital/startbootstrap-sb-admin-2.git",
     "fileMap": [
       {
-        "basePath": "dist",
+        "basePath": ".",
```

```suggestion
        "basePath": "",
```
Empty string is the same as `.`, but consistent with existing packags.